### PR TITLE
mf-day4 completed homework

### PR DESCRIPTION
**Pull request by melodyhawkins**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
+    <!--[if lt IE 9]>
+      <script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
+      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
+    <![endif]-->
+  </head>
+  <body>
+    <div class="wrapper black">
+      <div class="container white pad-40">
+        <div class="row pad-40">
```
Comment (rockygg) [inline index.html:25]: the columns in this row do not add up to 12

File: index.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+          <div class="col-xs-2 black height-80"></div>
+          <div class="col-xs-1 col-xs-offset-5 black height-20 margin-xs"></div>
+          <div class="col-xs-1 col-xs-offset-1 black height-20 margin-xs"></div>
+          <div class="col-xs-1 col-xs-offset-1 black height-20 margin-xs"></div>
+        </div><!--END ROW 1-->
+          <div class="row pad-40">
+          <div class="col-md-12 black height-450"></div>
+        </div><!--END ROW 2-->
+      <div class="row pad-40">
```
Comment (rockygg) [inline index.html:34]: the columns in this row do not add up to 12

File: index.html [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        <div class="col-xs-2 height-20 black"></div>
+      </div><!-- END ROW 3 -->
+      <div class="row pad-40">
```
Comment (rockygg) [inline index.html]: the columns in this row do not add up to 12

File: index.html [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+        <div class="col-sm-6 black height-160 margin-55"></div>
+        <div class="col-sm-2 col-sm-offset-1 black height-160 margin-55"></div>
+        <div class="col-sm-2 col-sm-offset-1 black height-160 margin-55"></div>
+      </div><!--END ROW 4-->
+      <div class="row pad-40">
```
Comment (rockygg) [inline index.html:43]: the columns in this row do not add up to 12

**Review by rockygg — CHANGES_REQUESTED** [review]
I've requested changes, let me know if you have questions

**Comment by melodyhawkins** [comment]
Hi Rocky,
I am confused, I am looking at my index.html and those meta tags do come
before the other comments in the head. Perhaps I am missing something.
Please help. I made a few minor changes with comments so I hope this helps.

ROW 1: My first row of columns add up to 12. Do we not include the offset
of 5, 1, and 1?

ROW 2: This is one row of col-md-12. Please let me know how it does not add
up to 12?

ROW 3: Thank you, I fixed the failure to add up to 12 by adding a
col-xs-offset-10 to the row.

ROW 4: The columns add up to 12 unless we don’t count offset columns.

ROW 5: The columns add up to 12 unless we don’t count offset columns.

I tried to create a new pull request for you to review but could not.
Please let me know how to do this. Thank you for taking the time to help
me, Rocky.

All the best,
Melody

**Review by rockygg — APPROVED** [review]
Awesome

**Pull request merged**